operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -93,6 +93,11 @@ spec:
 # modify the service mesh.
 #    ---
 #    view_only_mode: false
+#
+# When true, Kiali will start with arguments needed for system tests to generate
+# code coverage report. This expects the Kiali image to contain Kiali coverage binary.
+#    ---
+#    system_test_codecov_enabled: false
```

Comment:
again, this is not very good if we need to do this. This is the user-focused configuration file. Why are we putting internal system test things in here? Is there no other way to do this?